pontoon/base/static/js/fluent_interface.js

```diff
@@ -104,7 +104,15 @@ var Pontoon = (function (my) {
     return elements.every(function(element) {
       return (
         isSimpleElement(element) ||
-        (element.type === 'Placeable' && element.expression.type === 'SelectExpression')
+        (
+          element.type === 'Placeable' &&
+          element.expression.type === 'SelectExpression' &&
+          element.expression.variants.every(function(variant) {
+            return variant.value.elements.every(function(element) {
+              return isSimpleElement(element);
+            });
+          })
+        )
```

Comment:
This changes makes sure that we don't treat nested selectors as supported in rich editor?